### PR TITLE
[fix] mysql install on buster mysqld

### PR DESCRIPTION
**Pull request by bleuchtang**
Some fix to install alternc sql on Debian Buster.

Should be tested a lot ;)

### CODE REVIEW

**Comment by bleuchtang** [comment]
The DATETIME issue is the same as https://github.com/IHTSDO/snomed-database-loader/issues/19

**Comment by bleuchtang** [comment]
Note that with mariabd those problems does not appeared.

**Comment by kienanstewart** [comment]
https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sqlmode_no_zero_date

**Comment by kienanstewart** [comment]
MariaDB's changes to strict mode will not include no zero date (https://mariadb.com/kb/en/library/sql-mode/); and MySQL >=5.7 enables this by default.

I think we're better off focusing on supporting the default Debian mysql server for the moment. To avoid this issue, you can turn off no zero date or disable strict mode in MySQL 5.7

File: install/mysql.sql [hunk 1]
```diff
@@ -467,7 +467,7 @@ CREATE TABLE IF NOT EXISTS `domaines_type` (
     `name` VARCHAR (30) NOT NULL, -- Uniq name
     `description` TEXT, -- Human description
     `target` enum ('NONE', 'URL', 'DIRECTORY', 'IP', 'IPV6', 'DOMAIN', 'TXT') NOT NULL DEFAULT 'NONE', -- Target type
-    `entry` TEXT DEFAULT '', -- BIND entry
```
Comment (kienanstewart) [inline install/mysql.sql]: Cette changement n'a pas l'air d'être en lien avec la reste du PR. Serait-il possible de l'enlèver?

**Pull request closed without merging**